feature-libs/cart/base/components/cart-shared/cart-item-list/cart-item-list.component.spec.ts

```diff
@@ -297,6 +297,17 @@ describe('CartItemListComponent', () => {
       ).toBeDefined();
     });
 
+    it('should update controls when quantity change', () => {
+      fixture.detectChanges();
+      const mockItem0Qty = mockItem0.quantity;
+      mockItem0.quantity = 20;
+      component.items = [mockItem0, mockItem1];
+      expect(
+        component.form.controls[mockItem0.entryNumber].get('quantity')?.value
+      ).toEqual(20);
+      mockItem0.quantity = mockItem0Qty;
```

Comment:
👍  I understand we want to set back the quantity to the same it was before to not cause any leak for other future assertions for people, but first I need clarify[ this question before approving](https://github.com/SAP/spartacus/pull/15314/files#r824142667) (we might even remove this if you agree)